pandas/_libs/missing.pyx

```diff
@@ -354,10 +354,7 @@ class NAType(C_NAType):
         return NAType._instance
 
     def __repr__(self) -> str:
-        return "NA"
-
-    def __str__(self) -> str:
```

Comment:
Python will automatically use `__repr__` if one isn't defined, so this was redundant.